docs/sdk/clients.md

```diff
@@ -147,9 +147,19 @@ gaiacli send \
   --chain-id=<chain_id> \
   --name=<key_name> \
   --to=<destination_cosmosaccaddr> \
-  --generate-only
+  --generate-only > unsignedSendTx.json
 ```
 
+You can sign transactions file generated with `--generate-only` later on with the `sign` command and a key:
```

Comment:
"You can now sign the transactions file generated thought the `--generate-only` flag by providing your key to the following command:"

Reply:
Amended, thanks!